vavr/src/test/java/io/vavr/AssertionsExtensions.java

```diff
@@ -27,6 +27,7 @@ public static ClassAssert assertThat(Class<?> clazz) {
             this.clazz = clazz;
         }
 
+        @SuppressWarnings("deprecation")
```

Comment:
instead of this I would recommend using the new `canAccess` method, something like:
```java
public void isNotInstantiable() {
    try {
        Assertions.assertThat(clazz.getDeclaredConstructor().canAccess(null)).isFalse();
    } catch (NoSuchMethodException e) {
        throw new AssertionError("no default constructor found");
    }
}
```